<?xml version="1.0"?>
<launch>
  <include file="$(find sick_tim)/launch/sick_tim561_2.launch" />
  <node pkg="scalaser" type="LaserScanToPointCloud" name="my_cloud_1" args="/laser_1 /scan_1">
  <remap from="cloud" to="cloud_1"/>
  </node>

  <node pkg="scalaser" type="LaserScanToPointCloud" name="my_cloud_2"  args="/laser_2 /scan_2">
  <remap from="cloud" to="cloud_2"/>
  </node>

</launch>
